### DISCUSSION TITLE
카테고리 펼치기 질문입니다

### DISCUSSION THREAD

**Discussion by ge0nmo** [pronist/hELLO#131] [Ideas]
안녕하세요. 만들어 주신 스킨 잘 쓰고 있습니다.

한가지 원하는 점이 있다면 카테고리 펼치기가 유지 되었으면 합니다.

카테고리를 펼친  후 새로고침을 누르거나 이동을 하면 다시 접히는 데 의도하신 부분인지는 모르겠으나

펼친 후에 유지되는 기능이 있으면 좋을 것 같습니다.

감사합니다!

**Comment by pronist** [comment]
@ge0nmo 안녕하세요~! 

아이디어 감사드립니다. 
카테고리 펼치기 기능은 요청이 다소 있는 것 같아서 다음 번에 추가 할 예정입니다~ :)

**Reply by zineeworld** [reply]
@pronist 안녕하세요. 펼치기 기능 혹시 지금은 제공되나요? 제가 못 찾는 걸까요?
